CMakeLists.txt

```diff
@@ -141,3 +141,8 @@ add_custom_target(update-headers
         VERBATIM
         COMMAND ${SHELL_FOR_PYTHON} -c "python ./generate_header.py ${CMAKE_SOURCE_DIR}/include/can/core/ids.hpp --source-dir=${OPENTRONS_HARDWARE_PATH} --language=c++"
```

Comment:
note for the future: we should make this a nice poetry script now that it's in a language environment